tests/expected/uninit/fixme_unions.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Z uninit-checks
+
+//! Tests for handling potentially uninitialized memory access via unions.
+//! TODO: add a `.expected` file for this test.
+
+use std::ptr::addr_of;
+
+#[repr(C)]
+#[derive(Clone, Copy)]
+union U {
+    a: u16,
+    b: u32,
+}
+
+/// Reading padding data via simple union access if union is passed to another function.
+#[kani::proof]
+unsafe fn cross_function_union_should_fail() {
+    unsafe fn helper(u: U) {
+        let padding = u.b; // Read 4 bytes from `u`.
+    }
+    let u = U { a: 0 }; // `u` is initialized for 2 bytes.
+    helper(u);
+}
+
+/// Reading padding data but a union is via a union from behind a pointer.
```

Comment:
Cannot parse this comment :)

Reply:
Totally fair :) "via a union from" should not have been there